### ISSUE TITLE
Bigint support

### ISSUE THREAD

**Issue by Ajinkya1496** [brianc/node-postgres#1976]
I am using postgres 10 version, typeorm version 6.1.3 and pg version 7.11.0

I am facing issue with the bigint datatype. My entity looks like this.
```
@Entity({
  name:'users'
})
export class Users {
  @PrimaryGeneratedColumn()
  id: number;

  @Column()
  name: string;

  @Column()
  team_id: number;
}
```

Now when I convert team_id datatype to bigint it gives me error saying
`Data type "BigInt" in "Users.team_id" is not supported by "postgres" database`
Postgres supports bigint and now javascript also supports bigint datatype.

How do I use bigint here in the entity. Current behaviour is that it converts bigint to string and wherever in the database I am using team_id, it goes as string only.

Is there possible solution or alterations that I can do to fix this.

**Comment by vitaly-t** [comment]
This is the standard behavior, for all 64-bit integers to be presented as strings. See [this explanation](https://stackoverflow.com/questions/39168501/pg-promise-returns-integers-as-strings).

**Comment by Bessonov** [comment]
See https://github.com/brianc/node-pg-types/issues/78

**Comment by vitaly-t** [comment]
@brianc This should be closed as a duplicate of #78.

**Comment by brianc** [comment]
Closing as dupe of #78
